### PR TITLE
test(e2e): add additional test cases for components per reported bugs

### PR DESCRIPTION
**Pull request by maxym-valor**
## Related Issue(s)

closes none

## Description

Added additional test cases for combobox(platform) and table(core), inline help(core), date-time picker(platform), alert(core), table(core)

##### During Implementation

1. Visual Testing:

-   [n/a] visual misalignments/updates
-   [n/a] check Light/Dark/HCB/HCW themes
-   [n/a] RTL/LTR - proper rendering and labeling
-   [n/a] responsiveness(resize)
-   [n/a] Content Density (Cozy/Compact/(Condensed))
-   [n/a] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [n/a] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [n/a] Mouse vs. Keyboard support
-   [n/a] Text Truncation

2. API and functional correctness

-   [n/a] check for console logs (warnings, errors)
-   [n/a] API boundary values
-   [n/a] different combinations of components - free style
-   [n/a] change the API values during testing

3. Documentation and Example validations

-   [n/a] missing API documentation or it is not understandable
-   [n/a] poor examples
-   [n/a] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

##### PR Quality

-   [x] the commit message(s) follows the guideline:
        https://github.com/SAP/fundamental-ngx/blob/main/CONTRIBUTING.md
-   [x] tests for the changes that have been done
-   [x] all items on the PR Review Checklist are addressed :
        https://github.com/SAP/fundamental-ngx/wiki/PR-Review-Checklist
-   [n/a] Run npm run build-pack-library and test in external application
-   [n/a] update `README.md`
-   [n/a] [Breaking Changes Wiki](https://github.com/SAP/fundamental-ngx/wiki/Breaking-Changes)

##### PR Review

-   [n/a] visual misalignments/updates
-   [n/a] check Light/Dark/HCB/HCW themes
-   [n/a] RTL/LTR - proper rendering and labeling
-   [n/a] responsiveness(resize)
-   [n/a] Content Density (Cozy/Compact/(Condensed))
-   [n/a] States - hover/disabled/focused/active/on click/selected/selected hover/press state
-   [n/a] Interaction/Animation - open/close, expand/collapse, add/remove, check/uncheck
-   [n/a] Mouse vs. Keyboard support
-   [n/a] Text Truncation

2. API and functional correctness

-   [n/a] check for console logs (warnings, errors)
-   [n/a] API boundary values
-   [n/a] different combinations of components - free style
-   [n/a] change the API values during testing

3. Documentation and Example validations

-   [n/a] missing API documentation or it is not understandable
-   [n/a] poor examples
-   [n/a] Stackblitz works for all examples

4. Accessibility testing
5. Browser Testing - Edge, Safari, Chrome, Firefox

### CODE REVIEW

**Comment by Seamoo13** [comment]
please add the component names to the title.

**Review by nikvalor — APPROVED** [review]

**Review by Seamoo13 — APPROVED** [review]

File: e2e/wdio/core/pages/alert.po.ts [hunk 1]
```diff
@@ -7,6 +7,9 @@ export class AlertPo extends CoreBaseComponentPo {
 
     button = ' button';
     alerts = 'fd-alert-example .fd-alert';
+    playgroundAlert = '.fd-playground .fd-alert ';
```
Comment (N1XUS) [inline e2e/wdio/core/pages/alert.po.ts:10]: Do you need space at the end of the selector?

File: e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts [hunk 2]
```diff
@@ -237,17 +238,46 @@ describe('Datetime picker suite', () => {
...
+    // skipped due to https://github.com/SAP/fundamental-ngx/issues/7112
+    xit('should check that date-time picker does not have error if it contains valid value', () => {
+        scrollIntoView(inputGroup, 8);
+        let validDate;
+        const currentDate = new Date();
+        const currentMonth = currentDate.getMonth() + 1;
+        const currentDay = currentDate.getDate();
+        const currentYear = currentDate.getFullYear();
+        let currentMinute = (currentDate.getMinutes()).toString();
+        if (currentMinute === '0') {
```
Comment (N1XUS) [inline e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts:262]: I think this check will also pass in the next condition, so it can be removed here

Comment (maxym-valor) [reply e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts:262]: not needed remove condition, just my mistake, improved condition, thank you!!

File: e2e/wdio/core/pages/alert.po.ts (hunk 1 above)
Comment (maxym-valor) [reply e2e/wdio/core/pages/alert.po.ts:10]: yeah, in other case test can fail due to incorrect selector

Comment (N1XUS) [reply e2e/wdio/core/pages/alert.po.ts:10]: Ok

**Review by N1XUS — APPROVED** [review]

**Review by puru-hk — APPROVED** [review]

File: e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts [hunk 3]
```diff
@@ -237,17 +238,49 @@ describe('Datetime picker suite', () => {
...
+            currentMinute = '0' + currentMinute;
+        }
+        const currentHour = currentDate.toLocaleString('en-US', { timeZone: 'UTC', hour: 'numeric', hour12: true });
+
+        if (currentHour[1] === ' ') {
+            validDate = `${currentMonth}/${currentDay}/${currentYear}, ${
+                currentHour[0]
+            }:${currentMinute} ${currentHour.slice(2, 4)}`;
+        }
+        if (currentHour[1] !== ' ') {
```
Comment (N1XUS) [inline e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts:274]: You can use just `else`

**Review by nikvalor — APPROVED** [review]

**Review by Seamoo13 — APPROVED** [review]

File: e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts (hunk 3 above)
Comment (maxym-valor) [reply e2e/wdio/platform/tests/date-time-picker.e2e-spec.ts:274]: Had a conversation with @N1XUS and @Seamoo13 and decided to keep it as it is

**Pull request merged**